backend/account/src/main/java/com/htec/account/exception/ExceptionSupplier.java

```diff
@@ -48,9 +48,9 @@
 
     public static final Supplier<ShelfException> userIsNotLoggedIn = () -> new ShelfException(
             ErrorMessages.USER_IS_NOT_LOGGED_IN.getErrorMessage(),
-            HttpStatus.UNAUTHORIZED.value(),
+            HttpStatus.BAD_REQUEST.value(),
```

Comment:
I talked to Panta about this and he said that unauthorized should only be when access token is expired and not valid since they have made a filter regarding that on frontend

Reply:
I talked about this with Panta and he said that unauthoruzed status is returned only when jwt token is not valid or expired since they made some filters on frontend.

Reply:
fine